WinBuild.cmd

```diff
@@ -0,0 +1,43 @@
+@echo off
+@setlocal EnableDelayedExpansion
+
```

Comment:
OK, I'll move two files to another folder. Which do you prefer among Windows/Scripting/contrib as its name according to your policy?

The name "Scripting" is from the manner of BVLC/Caffe windows branch.